Jenkinsfile

```diff
@@ -154,11 +154,12 @@ pipeline {
                 stash name: 'repo-amd64',includes: 'ceph-amd64.tar,cockroachdb-amd64.tar,cassandra-amd64.tar,nfs-amd64.tar,yugabytedb-amd64.tar,build/common.sh,_output/tests/linux_amd64/,_output/charts/,tests/scripts/'
                 script {
                     def data = [
+                        "aws_1.11.x": "v1.11.10",
+                        "aws_1.13.x": "v1.13.12",
```

Comment:
should 1.12 and 1.14. be included also?

Reply:
This time I thought it would be sufficient to skip some of the older builds so we don't have such a huge test matrix. I don't think we've ever hit an issue in a K8s version that wasn't also hit in either the version before or after, so 1.12, 1.14, and 1.16 are skipped for now.